multi-libvterm.el

```diff
@@ -72,47 +72,44 @@ If this is nil, setup to environment variable of `SHELL'."
 ;;;###autoload
 (defun multi-libvterm (&optional _buffer-name)
```

Comment:
The arg came from this PR. I think we should have a docstring to explain why this arg here.